### PR TITLE
add support for environment without private network

### PR DESCRIPTION
**Pull request by lmayorga1980**

### CODE REVIEW

**Comment by mchllweeks** [comment]
@lmayorga1980 can you please follow our contributing doc which can be found here: https://github.com/puppetlabs/beaker/blob/master/CONTRIBUTING.md.  Please add a spec test so that we don't break this new functionality in the future.

File: lib/beaker/hypervisor/vagrant.rb [hunk 1]
```diff
@@ -48,7 +48,7 @@ def make_vfile hosts, options = {}
         v_file << "    v.vm.box_download_insecure = '#{host['box_download_insecure']}'\n" unless host['box_download_insecure'].nil?
         v_file << "    v.vm.box_check_update = '#{host['box_check_update'] ||= 'true'}'\n"
         v_file << "    v.vm.synced_folder '.', '/vagrant', disabled: true\n" if host['synced_folder'] == 'disabled'
-        v_file << private_network_generator(host)
+        v_file << private_network_generator(host) unless (host['private_network_disable'].nil? and host['private_network_disable'] == 'true' )
```
Comment (mchllweeks) [inline lib/beaker/hypervisor/vagrant.rb]: should the check here to add the private_network_generator to the v_file just look for whether host['private_network_disable'] is set to true?

Comment (lmayorga1980) [reply lib/beaker/hypervisor/vagrant.rb]: @mchllweeks I definitely need to add some test. :+1:

**Comment by lmayorga1980** [comment]
@mchllweeks Do you happen to know where can I fix this issue?

```
Failure/Error: require 'beaker-rspec'
Beaker::Host::CommandFailure:
  Host 'ossecagent-win-29' exited with 1 running:
   type C:\Windows\System32\drivers\etc\hosts
  Last 10 lines of output were:
  	sh: line 0: type: C:WindowsSystem32driversetchosts: not found
```
```ruby
HOSTS:
  mybox:
    roles:
      - agent
      - default
    platform: windows-2012_r2-amd64
    box: windows_2012_r2_virtualbox
    hypervisor: vagrant
    user: vagrant
    is_cygwin: false
CONFIG:
  type: foss
  log_level: verbose
```
The vagrant box was created using https://github.com/joefitzgerald/packer-windows/blob/master/windows_2012_r2.json

Any help will be much appreciated.

**Comment by kevpl** [comment]
@lmayorga1980 apologies for the delay in response here. I think for this level of change a spec test would be enough for us to ensure we don't break your work in the future.

If you'd like us to take more time to try to reproduce the situation you mentioned above, please provide the repo/directory you're in & the command you're actually running as well, & hopefully we'll have some time to take a look at reproducing that.

**Comment by mchllweeks** [comment]
Last comment still current.  If no response in a week, we will close due to inactivity.

**Comment by lmayorga1980** [comment]
apologies @mchllweeks but still interested in this feature. Need to add some tests for this and get back this week.

**Comment by kevpl** [comment]
@lmayorga1980 last comment still current.

**Comment by lmayorga1980** [comment]
Closing this `PR` since the need for this feature is not current anymore.

**Pull request closed without merging**